docs/index.rst

```diff
@@ -80,6 +80,7 @@ MONETIO please refer to:
    examples/airnow_wrfchem
    examples/airnow_camchem_se
    examples/camchem
+   examples/airnow_wrfchem_reg
```

Comment:
Maybe put it after airnow_wrfchem instead?

Reply:
Yes, we could move it there too. I'll move it when I fix Beiming's CSI plot label too. Will do this now.